cli/sandbox-resource.yaml

```diff
@@ -11,6 +11,10 @@ name: "@{dev}-@{team}-x-@{commit}"    # name of the sandbox
 spec:  
   cluster: staging                            # cluster in which this sandbox is to reside.
   description: "@{dev}'s work on feature-x"   # a short description
+  defaultRouteGroup:
```

Comment:
Here we are adding the defaultRouteGroup endpoint but we are not removing the old forked endpoint from bellow:

```yaml
    endpoints:                                # endpoints associated with this fork.
    - name: customer                          # name of the endpoint.
      protocol: http                          # protocol (http or grpc).
      port: 8081                              # port to which the endpoint will resolve.
```